functions/grayscale/src/lib.rs

```diff
@@ -1,9 +1,16 @@
 #![allow(unused_imports)]
 use wasmedge_bindgen::*;
 use wasmedge_bindgen_macro::*;
+
 mod grayscale;
 
 #[wasmedge_bindgen]
 pub fn grayscale(image_data: Vec<u8>) -> String {
     grayscale::grayscale_internal(&image_data)
 }
+
+#[wasmedge_bindgen]
+pub fn grayscale_str(image_data: String) -> String {
```

Comment:
grayscale is deprecated now, we may clean it up.

Reply:
Do you mean we only need `greyscale_str` and no longer need `greyscale`? Thanks

Reply:
I have a dig, it's still used at:
https://github.com/second-state/dapr-wasm/blob/d097c475c3a38bcb0a56139c866691714f05ea78/image-api-go/image_api.go#L50

There are two functions here because the parameter type is different, not other special reasons.

Reply:
Why not change both examples to use the same Wasm function? Thanks.

Reply:
Ok, will have a fix on that later.

Reply:
@DarumaDocker I just see you released wasmedge-bindgen-host 0.3.0, I updated it and find this error:

![image](https://user-images.githubusercontent.com/230646/175519340-15bf60ea-542e-40b3-a88c-05c6ff989c64.png)

0.2.1 is OK.

Reply:
@juntao I just cleanup to remove grayscale_str.
Go and Rust will use the same API in wasm file.